velox/docs/functions/spark/string.rst

```diff
@@ -179,6 +179,26 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT substring('Spark SQL', -10, 3); -- "Sp"
         SELECT substring('Spark SQL', -20, 3); -- ""
 
+.. spark:function:: substring_index(string, delim, count) -> varchar
+
+    Returns the substring from ``string`` before ``count`` occurrences of the delimiter ``delim``.
+    Here the 'string' can be VARCHAR or VARBINARY and return type matches type of 'string'.
+    If ``count`` is positive, everything to the left of the final delimiter
+    (counting from the left) is returned. If ``count`` is negative, everything to the right
+    of the final delimiter (counting from the right) is returned. If ``count`` is 0, empty string
```

Comment:
Would you clarify what happens if 'delim' is not found or found fewer times than 'count?